### ISSUE TITLE
Running ndlib-rest on Windows 10?

### ISSUE THREAD

**Issue by szymanskim** [GiulioRossetti/ndlib-rest#9]
Is it possible to run app on Windows 10?
I get empty UI while running prebuilt Docker image. 
![image](https://user-images.githubusercontent.com/23433957/41221312-0d2a9202-6d64-11e8-9818-b970216d6bbf.png)
Console logs points to node.js:
![image](https://user-images.githubusercontent.com/23433957/41221414-5f7f923c-6d64-11e8-8cb0-0e70915889d5.png)

When I am building the image by myself, it fails again
![image](https://user-images.githubusercontent.com/23433957/41222931-a8f24014-6d68-11e8-9ba1-93eedc91448c.png)

Local testing
![image](https://user-images.githubusercontent.com/23433957/41222581-9bcf990a-6d67-11e8-9263-96f7895a89e8.png)

Running gunicorn:
![image](https://user-images.githubusercontent.com/23433957/41222665-e218a35c-6d67-11e8-829b-fa6b1eb356b5.png)
fcntl looks like Linux command.

**Comment by GiulioRossetti** [comment]
I'll try to reproduce the docker error as soon as possible.

In the meantime check if your current installation of ndlib is updated: from your local testing screen it seems that your version does not have the SWIRModel class.

**Comment by szymanskim** [comment]
You're right. It was outdated, but I was sticking to the README.md file libraries versions (maybe the file should be updated ;))

Now, the server starts but nothing more.
![image](https://user-images.githubusercontent.com/23433957/41228509-e95d53e0-6d78-11e8-8f4f-084c821266e5.png)

Only sometimes throws:
![image](https://user-images.githubusercontent.com/23433957/41228617-46d8c374-6d79-11e8-923a-53916eea55a7.png)

It listens on :5000 but it does not respond
![image](https://user-images.githubusercontent.com/23433957/41228754-b95a503e-6d79-11e8-969c-e3d292b42dbe.png)

**Comment by GiulioRossetti** [comment]
Ok, it seems that - at least on Windows - there is an issue with the log file generation. It should not prevent the service from running though.

Remember that ndlib-rest by itself provide access only to the remote experiment server, not to the visual framework (which, still being in an early beta, is shipped separately). Once started ndrest.py you can access the REST api as specified in the documentation page: http://localhost:5000/docs 

If you want to run the viz service, once installed and executed ndlib-rest, you have to check-out and configure the following repository: https://github.com/GiulioRossetti/NDLib_viz

**Comment by szymanskim** [comment]
The problem is I cannot access the REST API, I got 404. If I understand you correctly, this URL should be working.
![image](https://user-images.githubusercontent.com/23433957/41230265-6ff2baac-6d7f-11e8-967c-1b286c9aa0ab.png)

**Comment by GiulioRossetti** [comment]
Quite strange...
Try to comment ndrest.py lines 52-53 and 77 to 82. Maybe is the logging service that for some reasons forbid the service to execute properly.


```
import logging
from logging.handlers import RotatingFileHandler
....

# Request Logging
logger = logging.getLogger('werkzeug')
handler = RotatingFileHandler('logs/access.log.gz', maxBytes=1000, backupCount=1)
handler.setLevel(logging.INFO)
logger.addHandler(handler)
app.logger.addHandler(handler)
```

**Comment by szymanskim** [comment]
I've tried and nothing new happened, unfortunately

**Comment by raginigupta6** [comment]
Hey, having the same problem. The URL throws 404. Any leads? Thanks in advance

**Comment by GiulioRossetti** [comment]
Hi, unfortunately, I don't have a Windows machine to reproduce the issue at the moment.
A workaround could be using the docker image instead of directly installing the rest service.
